src/test/java/io/stargate/sgv2/jsonapi/api/v1/HttpStatusCodeIntegrationTest.java

```diff
@@ -0,0 +1,118 @@
+package io.stargate.sgv2.jsonapi.api.v1;
+
+import static io.restassured.RestAssured.given;
+import static io.stargate.sgv2.common.IntegrationTestUtils.getAuthToken;
+import static org.hamcrest.Matchers.blankString;
+import static org.hamcrest.Matchers.endsWith;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.not;
+import static org.hamcrest.Matchers.notNullValue;
+
+import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.junit.QuarkusIntegrationTest;
+import io.restassured.http.ContentType;
+import io.stargate.sgv2.api.common.config.constants.HttpConstants;
+import io.stargate.sgv2.jsonapi.testresource.DseTestResource;
+import org.junit.jupiter.api.Test;
+
+@QuarkusIntegrationTest
+@QuarkusTestResource(DseTestResource.class)
+public class HttpStatusCodeIntegrationTest extends AbstractCollectionIntegrationTestBase {
+  @Test
+  public void unauthenticated() {
```

Comment:
Don't we have a set of those in integration tests for each resource? I think the idea was, general errors, validations, etc, are validated in the:

* `GeneralResourceIntegrationTest`
* `NamespaceResourceIntegrationTest`
* `CollectionResourceIntegrationTest`

for each endpoint we have..